sklearn/utils/extmath.py

```diff
@@ -569,6 +569,17 @@ def svd_flip(u, v, u_based_decision=True):
     return u, v
 
 
+def flip_max_to_positive(u):
```

Comment:
Please make that private. Also, it's likely to be only useful for the PLS-style models so it's better to put that helper in the pls module itself I think.